client/post-editor/editor-ground-control/index.jsx

```diff
@@ -312,6 +312,9 @@ export default React.createClass( {
 	},
 
 	render: function() {
+		const postDate = this.props.post && this.props.post.date
```

Comment:
Maybe worth adding a comment to avoid dropping it before reduxifying date updates?